src/Module/AcctLink.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Friendica\Module;
+
+use Friendica\BaseModule;
+use Friendica\Network\Probe;
+
+/**
+ * Redirects to another URL based on the parameter 'addr'
+ */
+class AcctLink extends BaseModule
```

Comment:
I guess the class mustn't be named this way due to limits of our current router. You have to name it ```Acctlink```

Reply:
Yes, you're right! It's because of 
https://github.com/friendica/friendica/blob/6b2a12319c85c3bbda74c55a239e55f1b86ea91c/index.php#L270
`ucfirst()`
=> will rename it